Corona-Warn-App/src/test/java/de/rki/coronawarnapp/bugreporting/censors/dccticketing/DccTicketingJwtCensorTest.kt

```diff
@@ -0,0 +1,57 @@
+package de.rki.coronawarnapp.bugreporting.censors.dccticketing
+
+import de.rki.coronawarnapp.dccticketing.core.transaction.DccTicketingValidationCondition
+import io.kotest.matchers.shouldBe
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.jupiter.api.Test
+import testhelpers.BaseTest
+
+@Suppress("MaxLineLength")
+class DccTicketingJwtCensorTest : BaseTest() {
+
+    private val rawJwt = "rawAccessToken"
+
+    private val vc = DccTicketingValidationCondition(
+        hash = "hash", // not censored
+        lang = "lang", // not censored
+        fnt = "family name",
+        gnt = "given name",
+        dob = "1981-03-20",
+        type = listOf("vaccination", "recovery"), // not censored
```

Comment:
should not it be `v`, `r`

Reply:
luckily this is not censored (censoring single letters would create chaos!)